go.mod

```diff
@@ -28,14 +28,15 @@ require (
 	github.com/openshift/ptp-operator v0.0.0-20240404165119-29a3d7b3d60b
 	github.com/operator-framework/api v0.23.0
 	github.com/operator-framework/operator-lifecycle-manager v0.28.0
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.72.0
```

Comment:
This version is too new for eco-goinfra because it based on api v0.30.1 Please use v0.73.2 instead

Reply:
@kononovn sorry, I did not understand your request: v.72.0 is older then  v0.73.2, no?

Reply:
done, but I still do not understand your point